builtins.go

```diff
@@ -1161,6 +1163,99 @@ func builtinParseJSON(i *interpreter, trace traceElement, str value) (value, err
 	return jsonToValue(i, trace, parsedJSON)
 }
 
+func builtinManifestJSONEx(i *interpreter, trace traceElement, obj, indent value) (value, error) {
```

Comment:
We have a very similar logic here https://github.com/google/go-jsonnet/blob/v0.16.0/interpreter.go#L695 and here: https://github.com/google/go-jsonnet/blob/v0.16.0/interpreter.go#L627. 

Could you add a TODO to look into unifying the two? (Unless you feel like doing that now.) 

Please also add a comment that we implement it instead of using a library function to have full control over formatting.

Reply:
I added a comment. I definitely think you are right, that these functions should be unified. I would have tackled this, but the use of `interface{}` in the above functions vs the `Value` based types might make it more difficult. I think that ValueArray and ValueObjects would need to be first converted to standard slice and map first, and iterated on again. I'm unsure if that's a significant performance impact or not.